.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Description
+Briefly describe the problem or reason for the changes implemented in the PR
+
+# Trac ticket
+Link to the Trac ticket for this PR (all PRs except really trivial issues or 
+tests adding need an accepted ticket).
```

Comment:
I think we should mention that you can write `ticket-<number>` and it’s [autolinked](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources).

```suggestion
# Trac ticket

<!-- Replace <number> with ticket number, or delete the line and write N/A if
this is a trivial PR. -->

ticket-<number>
```

Reply:
TIL! :heart_eyes: